app/assets/javascripts/mobile.coffee

```diff
@@ -2,6 +2,8 @@
 #= require _swipe
 
 [].forEach.call document.querySelectorAll('nav a'), (a) ->
+  a.className = ''
+  document.querySelector("nav a[href='" + location.pathname + "']").setAttribute('class', 'active')
```

Comment:
We should not place it here remove this line from the loop. We need to run it only once